source/_posts/2020-07-11-android-18-110-release.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "Home Assistant Companion for Android 1.8 - 1.10"
+description: "Today we're releasing a big update for our Android companion app including lots of improvements to notifications, biometrics support and a new geocoded sensor."
+date: 2020-06-18 00:00:00
+date_formatted: "June 18, 2020"
+author: Daniel Shokouhi
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2020-07-android-18-110-release/biometric_unlock.png
```

Comment:
A social image has recommended dimensions, putting in a vertical screenshot of a mobile device is going to look really weird. 

I can make OG image with the HA branding if you want?

Reply:
I was actually looking for the recommended image size but didn't get any luck.  If you could make that one would be great!!  I think if we crop out the blue part and just show the face unlock part would be good enough :)  What size should the image be?

Reply:
I was able to reduce the dimensions to 427x298 would that be good enough for the social image?

Reply:
Resolving as I think I got it? :)